otx/algorithms/common/adapters/mmcv/hooks.py

```diff
@@ -491,8 +491,8 @@ def _init_rule(self, rule, key_indicator):
         self.key_indicator = key_indicator
         self.compare_func = self.rule_map[self.rule]
 
-    def _should_check_stopping(self, runner):
-        """Called _should_check_stopping in ReduceLROnPlateauLrUpdaterHook."""
+    def _is_check_timing(self, runner):
```

Comment:
I think this name is a little confusing. how about using `time_to_use` or maybe what you think better?

Reply:
For the naming, I don't have any good idea. `time_to_use` also looks good to me. However, please put the return type.

```
def _is_check_timing(self, runner) -> bool:
```

Reply:
I also recommend adding type hints as well.

Reply:
Added type hint, but I love current name :) please go with as-is.